app/src/main/java/com/example/parkingsystemkotlin/mvp/model/reservation/ReservationInformationDB.kt

```diff
@@ -53,6 +55,23 @@ object ReservationInformationDB : DataBase {
         )
     }
 
+    override fun releasePastReservations(): Int {
+        var releasedReservations: Int = ZERO_INT
+        if (reservations.isNotEmpty()) {
+            reservations.forEach { reservation ->
+                releasedReservations += deleteReservations(reservation.key)
+            }
+        }
+        return releasedReservations
+    }
+
+    private fun deleteReservations(k: String): Int {
```

Comment:
use a more representative name instead of k

Reply:
done